proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -130,6 +130,37 @@ message OutputDebugInfo {
 
 message ModulesProgress {
   repeated ModuleProgress modules = 1;
+  repeated Job running_jobs = 2;
+  repeated ModuleStats modules_stats = 3;
+}
+
+message Job {
```

Comment:
If we move forward, we need to document all this field properly. That will be important especially on `ModuleStats` to correctly describe the "spec" and what each metric means exactly as well a potential gotchas.

Reply:
Still relevant for higher element in the proto, good job on the individual fields!